docs/DID Communications Research and Specification/i_credential-issuance.md

```diff
@@ -0,0 +1,79 @@
+# credential-issuance
+Creating an authenticated statement about a DID.
+
+The Verifiable Credential (VC) issuance flow consists of a three step interaction process between two parties, the <u>issuer</u> and the <u>holder</u>. This is the third interaction in this process. In this interaction, the <u>holder</u> asks the <u>issuer</u> for issuance of a specific VC.
```

Comment:
Perhaps mention and link to step 1 and 2.

Reply:
done, see 87482642